sdk/videoanalyzer/azure-media-videoanalyzer-edge/swagger/src/main/java/MethodRequestCustomizations.java

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+
+import com.azure.autorest.customization.Customization;
+import com.azure.autorest.customization.LibraryCustomization;
+import com.azure.autorest.customization.PackageCustomization;
+import com.azure.autorest.customization.ClassCustomization;
+
+public class MethodRequestCustomizations extends Customization {
```

Comment:
Since you have all the customization code in the swagger/README.md file, you don't need this java file and swagger/pom.xml file